ingestion_server/test/integration_test.py

```diff
@@ -457,3 +498,45 @@ def test_update_index_audio(self):
 
         # Wait for the task to send us a callback.
         assert self.__class__.cb_queue.get(timeout=120) == "CALLBACK!"
+
+    @pytest.mark.order(12)
```

Comment:
Out of curiosity, do you know why the order mark is needed for these? My understanding was that pytest ran tests in the order that they appear in the module. Is that different for class-method based tests? My understanding is confirmed by the `pytest-order` docs as well: https://pytest-dev.github.io/pytest-order/stable/intro.html#quickstart

Reply:
My memory is also fuzzy in that regards but, as I recall, Pytest was running them alphabetically. It'll be a simple change to remove the order and see if the tests fail (I'll do that in a separate PR after merging this).